.merlin

```diff
@@ -1,13 +1,19 @@
 ####{BSB GENERATED: NO EDIT
```

Comment:
Does this file have to be checked in? It contains some local configuration. Let's delete it if possible.